regimes/it/schemes.go

```diff
@@ -0,0 +1,91 @@
+package it
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/tax"
+)
+
+// Base scheme keys for this regime.
+const (
+	SchemeKeyOrdinary  cbc.Key = "ordinary"
+	SchemeKeyFreelance cbc.Key = "freelance"
+)
+
+var schemes = []*tax.Scheme{
```

Comment:
What do you think about exporting these? In order to access available schemes from the provider, I'd have to get the `Regime` instance from Italy. Might be simpler if we can do `it.Schemes`? Same for `it.Zones` and `it.Categories`